### PR TITLE
fix #52288 hasepost.de

### LINKED ISSUE #52288: hasepost.de
### Issue URL (Ads)
 [https://www.hasepost.de/staedtetag-verlangt-strategie-fuer-stufenweise-rueckkehr-zum-alltag-181321/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.hasepost.de%2Fstaedtetag-verlangt-strategie-fuer-stufenweise-rueckkehr-zum-alltag-181321%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.3
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Block third-party Authorization header, Self-destructing first-party cookies (4320), Self-destructing third-party cookies (180), Disable cache for third-party requests, Block Push API, Block Location API, Hide your Referrer from third-parties (www.hasepost.de), Hide your User-Agent ()
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy, <br/>Fanboy's Enhanced Tracking List<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Adblock Warning Removal List<br/><br/><b>Language-specific:</b><br/> AdGuard German
Custom filter subscriptions: |

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#52288

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -90,6 +90,8 @@ sodapdf.com##.be-tooltip
 !
 tube8.es###catFishBanner
 tube8.es###announcement_ticker
+hasepost.de#$#.pum-overlay { display: none!important; }
+hasepost.de#$#html { overflow: visible!important; }
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/popups.txt]: Changing it to .pum-open-overlay instead of html.

File: GermanFilter/sections/specific.txt [hunk 2]
```diff
@@ -67,6 +67,8 @@ haz.de,ostsee-zeitung.de,sn-online.de,dnn.de,kn-online.de,ln-online.de,neuepress
 !
 radiosaw.de###oms_gpt_rectanglesidebar
 fitbook.de##a[href^="https://secure.adnxs.com/"]
+||hasepost.de/wp-content/banners/
```
Comment (BlazDT) [inline GermanFilter/sections/specific.txt]: ||hasepost.de/wp-content/banners^

File: GermanFilter/sections/specific.txt [hunk 3]
```diff
@@ -67,6 +67,8 @@ haz.de,ostsee-zeitung.de,sn-online.de,dnn.de,kn-online.de,ln-online.de,neuepress
...
+||hasepost.de/fpassage/aktuell.gif
```
Comment (BlazDT) [inline GermanFilter/sections/specific.txt]: Not shown.

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: GermanFilter/sections/specific.txt (hunk 2 above)
Comment (Sergey-Lyapin) [reply GermanFilter/sections/specific.txt]: If I'm not wrong both rules work the same. Please explain why we need `^` in the rule?

File: GermanFilter/sections/specific.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply GermanFilter/sections/specific.txt]: for example <a href="https://www.hasepost.de/zahl-der-aktuell-corona-infizierten-region-osnabrueck-ruecklaeufig-181882/">here</a>
<details>
<summary>Screenshot</summary>

</details>

File: GermanFilter/sections/specific.txt (hunk 2 above)
Comment (Alex-302) [reply GermanFilter/sections/specific.txt]: `^` is unnecessary here. It is useful if url contains separators 
https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#special-characters

File: AnnoyancesFilter/sections/popups.txt (hunk 1 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/popups.txt]: this is ok

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**